src/axios/apiQueries.js

```diff
@@ -21,3 +21,16 @@ export const getCommentForSingleReview = (review_id) => {
     return commentsResponse.data.comments;
   });
 };
+
+export const getUpdateVotes = (review_id, inc_votes) => {
```

Comment:
The name of this function seems misleading. It actually updates the votes, not just getting an update that's already happened. Rename this to something clearer please